packages/react-router/src/utils.ts

```
@@ -4,7 +4,7 @@ import * as React from 'react'
 // React 18 with Webpack, which statically analyzes imports and fails when it
 // sees React.use referenced (since 'use' is not exported from React 18).
 // This uses a dynamic string lookup to avoid the static analysis.
-const REACT_USE = 'use'
+let REACT_USE = 'use'
```

ESLint

[error] 'REACT_USE' is never reassigned. Use 'const' instead.

(prefer-const)